src/Pages/EventList/services/getFilteredEventList.js

```diff
@@ -28,7 +30,16 @@ export function getFilteredEvents(
     eventFilter === 'Hidden' && filteredHiddenEvents.length > 0
       ? filteredHiddenEvents
       : eventFilter === 'Old'
-      ? filteredOldEvents
-      : filteredNotHiddenEvents
-  return filteredEvents
+        ? filteredOldEvents
+        : filteredNotHiddenEvents
+  console.log('chosenEventListFilter', chosenEventListFilter)
+  const filteredEventList =
+    chosenEventListFilter === 'Private' && filteredEvents.length > 0
+      ? filteredEvents.filter((filteredEvent) => filteredEvent.visibility === 'private')
+      : chosenEventListFilter === 'Favorite'
+        ? filteredEvents.filter((filteredEvent) => filteredEvent.favorite === true)
+        : filteredEvents.filter((filteredEvent) => filteredEvent.visibility === 'public')
```

Comment:
This is very difficult to understand. I would avoid using `?` multiple times in an assignment. 

[A funny article which might be of use](https://javascript.info/ninja-code)